packages/webdriverio/src/commands/element/scrollIntoView.ts

```diff
@@ -56,19 +56,20 @@ export async function scrollIntoView (
 
     let deltaX = 0
     let deltaY = 0
+    let origin = this
     /**
      * by default the WebDriver action scrolls the element just into the
-     * viewport. In order to stay complaint with `Element.scrollIntoView()`
+     * viewport. In order to stay compliant with `Element.scrollIntoView()`
      * we need to adjust the values a bit.
      */
     if (typeof options === 'boolean' || typeof options.block === 'string' || typeof options.inline === 'string') {
-        const htmlElem = await browser.$('html')
-        const viewport = await htmlElem.getSize()
+        origin = await browser.$('html')
+        const viewport = await origin.getSize()
         const elemSize = await this.getSize()
         if (options === true || (options as ScrollIntoViewOptions).block === 'start') {
-            deltaY += viewport.height - elemSize.height
+            deltaY += viewport.width - elemSize.width
```

Comment:
Can you explain how to determine the vertical delta when looking at the `wdith`of the element?

Reply:
Unless I'm missing something, but from my understanding the vertical delta is computed in the `deltaX` variable in the following lines

Here it was computing the horizontal delta but based on the `height` which doesn't make sense

Reply:
My bad, I meant to update the `deltaX`
They were both based the `height`, I will push an update soon